testing/sdk_auth.py

```diff
@@ -475,6 +424,10 @@ def cleanup(self) -> None:
             log.info("Deleting temporary working directory")
             self._temp_working_dir.cleanup()
 
+        # Delete secret
```

Comment:
Is the comment wrong, or the subsequent call, because the latter does not look like it's deleting a secret?

Reply:
This is referring to the service account secret, but indeed the comment it's redundant.